client/src/docs-ui/select-anchor/reroute-if-necessary.worker.ts

```diff
@@ -0,0 +1,21 @@
+import {parseURL} from "../../utils/url/url.worker";
+
+const rereouteCache = new Map<string, string>();
+
+export const rerouteIfNecessary = async (path: string) => {
```

Comment:
This name is a bit ambiguous. Can we reword this?

Reply:
Got rid of it altogether